packages/filesystem/src/node/encoding.ts

```diff
@@ -0,0 +1,48 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
```

Comment:
Same as the comment above.

Reply:
@vince-fugnitto I removed the "Copyright" line and replaced it with my name and e-mail, nevertheless not sure whether it's the right way to correct it. Any advice?

Reply:
It looks fine to me :)